Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -6496,4 +6496,126 @@ final class BuildPlanTests: XCTestCase {
             )
         }
     }
+
+    func testTraversalOverModules() async throws {
```

Comment:
Would you mind moving these two new test functions into a new `BuildPlanTraversalTests` test case? `BuildPlanTests` is already huge, and XCTest can only run separate `XCTestCase` subclasses in parallel, but not separate functions.

Reply:
Sure!

Reply:
✅